lib/puppet_x/jenkins.rb

```diff
@@ -1,2 +0,0 @@
-module PuppetX; end
-module PuppetX::Jenkins; end
```

Comment:
Not entirely, however it just cuts out the required number of bodged imports and number of files they need to be made in.